doc/cephfs/client-auth.rst

```diff
@@ -10,6 +10,13 @@ to the lowest possible level of authority needed.
     Path restriction and layout modification restriction are new features
     in the Jewel release of Ceph.
 
+.. note::
```

Comment:
I think move this note to createfs.rst would be better. The user would want to know this before creating the file system.

Reply:
createfs.rst already has a note about it.